client/api/eventsApi.js

```diff
@@ -12,23 +19,38 @@ export function addNewEvent (id, event) {
     .catch((error) => console.log(error))
 }
 
+/**
+ * Request array of events from projects table
+ * @returns {array} All events
```

Comment:
💭 We could introduce a [`@typedef`](https://jsdoc.app/tags-typedef.html), so that we can document the properties of an event object.  Note that `@typedef` has to be declared in its own docblock, i.e.:

```
/**
 * @typedef {object} Event
 * @property {number} id
 * ...
 */
/**
 * Request array of events from projects table.
 * @returns {Promise.<Event[]>}
 */
```